src/main/java/com/libertymutualgroup/herman/aws/ecs/EcsPush.java

```diff
@@ -224,9 +223,6 @@ public EcsPush(EcsPushContext context) {
     }
 
     public void push() {
-        StopWatch stopWatch = new StopWatch();
```

Comment:
Why remove the timer?

Reply:
@c1phr was working on removing any Spring deps there for the CLI version. I added this is a while back when playing around with reporting for Herman deploys over time for a project, but this isn't being used atm.